templates/statistics/position.html.twig

```diff
@@ -0,0 +1,196 @@
+                    <option value='cmp_02' {% if tri is same as('cmp_02') %}selected='selected'{% endif %}>Etage</option>
+                    <option value='cmp_03' {% if tri is same as('cmp_03') %}selected='selected'{% endif %}>Obligatoire</option>
+                    <option value='cmp_03desc' {% if tri is same as('cmp_03desc') %}selected='selected'{% endif %}>Renfort</option>
+                    <option value='cmp_2' {% if tri is same as('cmp_2') %}selected='selected'{% endif %}>Heures du - au +</option>
+                    <option value='cmp_2desc' {% if tri is same as('cmp_2desc') %}selected='selected'{% endif %}>Heures du + au -</option>
+                  </select>
+                </td>
+              </tr>
+              <tr style='vertical-align:top'>
+                <td><label class='intitule'>Postes</label></td>
+                <td>
+                  <select name='postes[]' multiple='multiple' size='20' onchange='verif_select("postes");' class='ui-widget-content ui-corner-all' >
+                    {% if postes_list is not empty %}
+                      <option value='Tous'>Tous</option>
+                        {% for elem in postes_list %}
+                          {% if postes is defined %}
+                            {% if elem.obligatoire == "Obligatoire" %}
+                              {% if elem.id in postes %}
+                                <option value='{{ elem.id }}' selected="selected" class='td_obligatoire'> {{ elem.nom | raw}} ({{ elem.etage | raw }})</option>
+                              {% else %}
+                                <option value='{{ elem.id }}' class='td_obligatoire'>{{ elem.nom | raw}} ({{ elem.etage | raw }})</option>
+                              {% endif %}
+                            {% else %}
+                              {% if elem.id in postes %}
+                                <option value='{{ elem.id }}' selected="selected" class='td_renfort'> {{ elem.nom | raw}} ({{ elem.etage | raw }})</option>
+                              {% else %}
+                                <option value='{{ elem.id }}' class='td_renfort'>{{ elem.nom | raw}} ({{ elem.etage | raw }})</option>
+                              {% endif %}
+                            {% endif %}
+                          {% endif %}
+                        {% endfor %}
+                    {% endif %}
+                  </select>
+                </td>
+              </tr>
+              {% if nbSites>1 %}
+                <tr style='vertical-align:top'>
+                  <td><label class='intitule'>Sites</label></td>
+                  <td>
+                    <select name='selectedSites[]' multiple='multiple' size='{{ nbSites+1 }}' onchange='verif_select("selectedSites");' class='ui-widget-content ui-corner-all' >
+                      <option value='Tous'>Tous</option>
+                        {% for i in 1..nbSites %}
+                          {% if i in selectedSites %}
```

Comment:
Site names are shifted in the dropdown menu.
- There is a blank option linked to the site 1
- The second option with the name of site 1 is linked to site 2
- ...

Reply:
solved with this commit 1b606b802939cc354ec73244cee8708f259c99ff